### ISSUE TITLE
[16216] Failed to execute script main

### ISSUE THREAD

**Issue by patrickgisherwood** [pyinstaller/pyinstaller#5254]
I am trying to compline a python project with Pyinstaller and this is my error that I am getting.   I tried taking matplotlib out of the script but I am still getting this same error.  My command to run pyinstaller was:  pyinstaller --onefile main.py

If someone could give me some insight into how I could go about resolving this I would appreciate it as I am fairly confused with how to go about debugging this.  Apparently, Pyinstaller is fairly straightforward but I must be missing something because I am quite confused on how to debug or use.  

MatplotlibDeprecationWarning: Matplotlib installs where the data is not in the mpl-data subdirectory of the package are deprecated since 3.2 and support for them will be removed two minor releases later.
  exec(bytecode, module.__dict__)
Traceback (most recent call last):
  File "main.py", line 3, in <module>
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "c:\users\patri\appdata\local\programs\python\python38\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 493, in exec_module
    exec(bytecode, module.__dict__)
  File "utils.py", line 3, in <module>
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "c:\users\patri\appdata\local\programs\python\python38\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 493, in exec_module
    exec(bytecode, module.__dict__)
  File "matplotlib\__init__.py", line 898, in <module>
  File "matplotlib\cbook\__init__.py", line 480, in _get_data_path
  File "matplotlib\__init__.py", line 239, in wrapper
  File "matplotlib\__init__.py", line 534, in get_data_path
  File "matplotlib\__init__.py", line 239, in wrapper
  File "matplotlib\__init__.py", line 566, in _get_data_path
RuntimeError: Could not find the matplotlib data files
[16216] Failed to execute script main

**Comment by rokm** [comment]
What version of `matplotlib` are you using? Currently, 3.3.x series is not supported, so if you're using that, you'll need to downgrade it to 3.2.x.

Or alternatively, you can try using the branch from PR #5006 which adds support for 3.3.x, by installing pyinstaller via:
```
pip install -U https://github.com/bwoodsend/pyinstaller/archive/fix-matplotlib.zip
```

**Comment by rokm** [comment]
This should be fixed since PyInstaller 4.1 with #5006 that added support for matplotlib 3.3.x series.
